front/services/api/ReqwestApiClient.js

```diff
@@ -158,6 +158,25 @@ export default class ReqwestApiClient {
         });
     }
 
+    unregisterEvent(eventSlug, token, callback) {
```

Comment:
File is named "ReqwestApiClient", should be RequestApiClient no ?

Reply:
Yes, I catch that, I propose to change this in another PR WDYT ?